core/examples/luigi-sample-angular/src/app/project/project.component.html

```diff
@@ -6,6 +6,12 @@
 </p>
 <div *ngIf="projectId && projectId !== 'overview.html'">
   LuigiClient navigation methods:<br>
+
+  <div *ngIf="preservedViewCallbackContext">
```

Comment:
Please improve styling for it. I would recommend a new section after "LuigiClient navigation methods"
![screen shot 2018-08-01 at 15 26 27](https://user-images.githubusercontent.com/8238188/43526981-c3acf556-95a5-11e8-9f07-9d07f2bbbcbe.png)

Reply:
This is being taken care of already in fundamental ui branch.